scripts/upgrade_beeper_room.py

```diff
@@ -194,6 +205,14 @@ def make_url(part: str):
     },
 )
 
+# Tombstone the old room and make it so that you can't invite or send events to it
```

Comment:
Need to do this after fixing permissions for the previous room, so that I'm allowed to do it even though I'm not a room admin. For some reason synapse allows me to raise my own power level and then issue the tombstone, but it doesn't allow me to just issue the tombstone if I'm not already a room admin.